### PR TITLE
Binding Template

### PR DESCRIPTION
**Pull request by Hipolyto**
Binding Template

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,347 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (thisisthekap) [inline .gitignore:1]: Remove `.DS_Store`, add it to `.gitignore`

File: LICENSE [hunk 2]
```diff
@@ -1,21 +0,0 @@
-MIT License
```
Comment (thisisthekap) [inline LICENSE]: Why was this file deleted?

File: README.md [hunk 3]
```diff
@@ -1 +0,0 @@
-# Xamarin.FacebookSlim.iOS
```
Comment (thisisthekap) [inline README.md:1]: Why was this file deleted?

File: Xamarin.FacebookSlim.Sample/Info.plist [hunk 4]
```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDisplayName</key>
+	<string>Nuke Sample</string>
```
Comment (thisisthekap) [inline Xamarin.FacebookSlim.Sample/Info.plist]: ?

File: Xamarin.FacebookSlim.Sample/ViewController.cs [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+            };
+
+            var skiaView = new SKCanvasView
+            {
+                TranslatesAutoresizingMaskIntoConstraints = false
+            };
+            skiaView.PaintSurface += OnPaintSurface;
+            Add(skiaView);
+
+            UIButton button2 = AddButton("Streamed image");
```
Comment (thisisthekap) [inline Xamarin.FacebookSlim.Sample/ViewController.cs]: is this an artifact form the copied nuke bindings sample?

File: Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj [hunk 6]
```diff
@@ -0,0 +1,73 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
+    <IsBindingProject>true</IsBindingProject>
+    <NoBindingEmbedding>true</NoBindingEmbedding>
+    <RootNamespace>Xamarin.FacebookSlim</RootNamespace>
+    <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.0</SupportedOSPlatformVersion>
+    <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-maccatalyst'">14.0</SupportedOSPlatformVersion>
```
Comment (thisisthekap) [inline Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj]: We do not support catalyst.

File: Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+    <BindingPlatform>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</BindingPlatform>
+    <AssemblyName>Xamarin.FacebookSlim</AssemblyName>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <Copyright>Xamarin FacebookSlim 2022</Copyright>
+    <PackageProjectUrl>https://github.com/thisisthekap/Xamarin.FacebookSlim.iOS</PackageProjectUrl>
+    <PackageIcon></PackageIcon>
+    <Authors></Authors>
+    <Owners></Owners>
```
Comment (thisisthekap) [inline Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj]: fun.music IT GmbH

File: Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+    <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-maccatalyst'">14.0</SupportedOSPlatformVersion>
+    <BindingPlatform>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</BindingPlatform>
+    <AssemblyName>Xamarin.FacebookSlim</AssemblyName>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <Copyright>Xamarin FacebookSlim 2022</Copyright>
+    <PackageProjectUrl>https://github.com/thisisthekap/Xamarin.FacebookSlim.iOS</PackageProjectUrl>
+    <PackageIcon></PackageIcon>
+    <Authors></Authors>
```
Comment (thisisthekap) [inline Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj]: fun.music IT GmbH, Hipolyto Obeso Huerta

File: Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <MtouchLink>SdkOnly</MtouchLink>
+    <MtouchHttpClientHandler>HttpClientHandler</MtouchHttpClientHandler>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <MtouchLink>SdkOnly</MtouchLink>
+    <MtouchHttpClientHandler>HttpClientHandler</MtouchHttpClientHandler>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
```
Comment (thisisthekap) [inline Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj]: why is this needed?

File: ios/FacebookProxy/FacebookProxy.h [hunk 10]
```diff
@@ -0,0 +1,12 @@
+//
+//
+
+#import <Foundation/Foundation.h>
+
+FOUNDATION_EXPORT double FacebokProxyVersionNumber;
```
Comment (thisisthekap) [inline ios/FacebookProxy/FacebookProxy.h]: Typo?

File: ios/FacebookProxy/FacebookProxy.swift [hunk 11]
```diff
@@ -0,0 +1,227 @@
+//
+
+import Foundation
+import UIKit
+import FacebookLogin
+
+@objc public enum loginResultEnum : Int {
```
Comment (thisisthekap) [inline ios/FacebookProxy/FacebookProxy.swift]: casing?

File: ios/FacebookProxy/FacebookProxy.swift [hunk 12]
```diff
@@ -0,0 +1,227 @@
...
+    case   UIControlErrors
+    case   GraphAPIDebugWarning
+    case   GraphAPIDebugInfo
+    case   NetworkRequests
+    case   DeveloperErrors
+}
+
+@objc public class LoginResult: NSObject {
+
+   @objc public var token = ""
```
Comment (thisisthekap) [inline ios/FacebookProxy/FacebookProxy.swift]: formatting

**Review by thisisthekap — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Hipolyto) [reply .gitignore:1]: added

File: LICENSE (hunk 2 above)
Comment (Hipolyto) [reply LICENSE]: was added back

File: README.md (hunk 3 above)
Comment (Hipolyto) [reply README.md:1]: is added back

File: Xamarin.FacebookSlim.Sample/ViewController.cs (hunk 5 above)
Comment (Hipolyto) [reply Xamarin.FacebookSlim.Sample/ViewController.cs]: removed

File: Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj (hunk 8 above)
Comment (Hipolyto) [reply Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj]: removed

File: Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj (hunk 9 above)
Comment (Hipolyto) [reply Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj]: removed

File: ios/FacebookProxy/FacebookProxy.swift (hunk 11 above)
Comment (Hipolyto) [reply ios/FacebookProxy/FacebookProxy.swift]: done

File: README.md (hunk 3 above)
Comment (thisisthekap) [reply README.md:1]: ???

File: License [hunk 13]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 thisisthekap
+Copyright (c) 2020 Jean-Marie Alfonsi
```
Comment (thisisthekap) [inline License]: ???

**Review by thisisthekap — APPROVED** [review]
